getting_started/workflow/export/exporting_for_web.rst

```diff
@@ -12,27 +12,11 @@ in the user's browser.
                with :kbd:`F12`, to view **debug information** like JavaScript,
                engine, and WebGL errors.
 
-.. attention:: Many browsers, including Firefox and Chromium-based browsers,
-               will not load exported projects when **opened locally** per
-               ``file://`` protocol. To get around this, use a local server.
-
-               .. tip:: Python offers an easy method to start a local server.
-                        Use ``python -m http.server 8000 --bind 127.0.0.1`` with Python 3 to serve the
-                        current working directory at ``http://localhost:8000``.
-                        `Refer to MDN for additional information <https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server>`__.
-
-.. attention:: `There are significant bugs when running HTML5 projects on iOS <https://github.com/godotengine/godot/issues/26554>`__
+.. attention:: `There are significant bugs when running HTML5 projects on iOS <https://github.com/godotengine/godot/issues?q=is:issue+is:open+label:platform:html5+ios>`__
                (regardless of the browser). We recommend using
                :ref:`iOS' native export functionality <doc_exporting_for_ios>`
                instead, as it will also result in better performance.
 
-.. note::
-
-    If you use Linux, due to
-    `poor Firefox WebGL performance <https://bugzilla.mozilla.org/show_bug.cgi?id=1010527>`__,
-    it's recommended to play the exported project using a Chromium-based browser
-    instead of Firefox.
```

Comment:
Did you intend to remove this note?
It physically hurts me to recommend using Chrome on Linux, but I think it's sadly still true (though I resist and still use FF to play Godot web exports :P).

Reply:
That was probably a failed merge on my part. I'll re-check the PR myself.
That said, on the specific topic, as we have seen with the recent regression, suggesting using Chrome right now will give a very bad impression if the game has any audio.

Which is why, in general, I don't like we taking the role of evaluating browsers "performances", and I'd rather point user to feature supports (e.g. can I use/mdn docs), and point out mileage might vary with different browsers, pointing maybe to "lead browsers" bug trackers.